extensions/amp-access/0.1/amp-access-server.js

```diff
@@ -180,6 +193,39 @@ export class AccessServerAdapter {
     return this.clientAdapter_.pingback();
   }
 
+  /** @override */
+  encodeReaderId(readerId) {
+    return this.getSubtleWithKey_().then(key => {
+      return this.subtle_.encrypt(key, readerId).then(s => {
+        dev.error(TAG, "bernie: encodeReaderId(): ", s);
```

Comment:
1. Why `dev.error`? Are you just trying to debug?
2. Very important! If you override `then`, you must return the value. E.g. you are missing `return s;`